### DISCUSSION TITLE
Date Picker的default-value和default-time区别是什么？

### DISCUSSION THREAD

**Discussion by nieyuyao** [element-plus/element-plus#6676] [Q&A]
贴下文档中的解释
<img width="816" alt="image" src="https://user-images.githubusercontent.com/17698194/158590674-6d2c1a8d-a31c-41cc-8676-e07a2f94dc95.png">

疑问是：
default-time是不是指的只是时刻，不包含年月日等日期部分？如果是的话，那么对于Date Picker是不是不应该有default-time选项，因为该组件无法设置时刻？同样的，如果是的话，对于可以设置时刻的组件DateTime Picker组件，同时设置了default-value和default-time，那么真正用来作为默认时间的将是default-value+default-time么？

**Comment by iamkun** [comment]
When you are picking a date on the picker, the target Date-time will be the 'date' you've picked as "YYYY-MM-DD" + the 'default time' as "HH:mm:ss"
